errtypes/errors.go

```diff
@@ -58,9 +58,9 @@ type RefreshError struct {
 
 func (e *RefreshError) Error() string {
 	if e.Err == nil {
-		return fmt.Sprintf("Server error: %v", e.genericError)
+		return fmt.Sprintf("Refresh error: %v", e.genericError)
```

Comment:
These are changes that should be in a separate PR.

Reply:
:+1: